.github/CODEOWNERS

```
@@ -0,0 +1,57 @@
+detectors/gcp/                                                          @open-telemetry/go-approvers
+
+instrumentation/github.com/astaxie/beego/otelbeego/                     @open-telemetry/go-approvers
+instrumentation/github.com/aws/aws-lambda-go/otellambda/                @open-telemetry/go-approvers
+instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/                   @open-telemetry/go-approvers
+instrumentation/github.com/bradfitz/gomemcache/memcache/otelmemcache/   @open-telemetry/go-approvers
+instrumentation/github.com/emicklei/go-restful/otelrestful/             @open-telemetry/go-approvers
+instrumentation/github.com/gin-gonic/gin/otelgin/                       @open-telemetry/go-approvers
+instrumentation/github.com/go-kit/kit/otelkit/                          @open-telemetry/go-approvers
+instrumentation/github.com/gocql/gocql/otelgocql/                       @open-telemetry/go-approvers
+instrumentation/github.com/gorilla/mux/otelmux/                         @open-telemetry/go-approvers
+instrumentation/github.com/labstack/echo/otelecho/                      @open-telemetry/go-approvers
+instrumentation/github.com/Shopify/sarama/otelsarama/                   @open-telemetry/go-approvers
+instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/            @open-telemetry/go-approvers
+instrumentation/google.golang.org/grpc/otelgrpc/                        @open-telemetry/go-approvers
+instrumentation/gopkg.in/macaron.v1/otelmacaron/                        @open-telemetry/go-approvers
+
+instrumentation/host/                                                   @open-telemetry/go-approvers
+instrumentation/net/http/httptrace/otelhttptrace/                       @open-telemetry/go-approvers
+instrumentation/net/http/otelhttp/                                      @open-telemetry/go-approvers
+instrumentation/runtime/                                                @open-telemetry/go-approvers
+
+propagators/autoprop/                                                   @open-telemetry/go-approvers
+propagators/aws/                                                        @open-telemetry/go-approvers
+propagators/b3/                                                         @open-telemetry/go-approvers
+propagators/jaeger/                                                     @open-telemetry/go-approvers
+propagators/opencensus/                                                 @open-telemetry/go-approvers
+propagators/ot/                                                         @open-telemetry/go-approvers
+
+samplers/aws/xray/                                                      @open-telemetry/go-approvers
+samplers/jaegerremote/                                                  @open-telemetry/go-approvers
+samplers/probability/consistent/                                        @open-telemetry/go-approvers
+
+zpages/                                                                 @open-telemetry/go-approvers
```

```suggestion
zpages/                                                                 @open-telemetry/go-approvers @open-telemetry/collector-maintainers
```

cc @open-telemetry/collector-maintainers